media/js/src/simulations/simulation4/realDataPolynomials.jsx

```diff
@@ -56,13 +56,11 @@ export const RealDataPolynomials = ({
 
     useEffect(() => {
         if (selectedOption === 0) {
-            setIncorrectFeedback2('Linear regression is not the best fit for ' +
-                'this dataset. This is evident when we run the tests against ' +
-                'Quadratic and then Cubic regressions.');
+            setIncorrectFeedback2('This is evident when we run the Linear ' +
+                'test against Quadratic, and then Cubic Regressions.');
         } else if (selectedOption === 1) {
-            setIncorrectFeedback2('Quadratic regression is not the best fit ' +
-                'for this dataset. This is evident when we run the tests ' +
-                'against Linear and then Cubic regressions.');
+            setIncorrectFeedback2('This is evident when we run the Quadratic ' +
```

Comment:
The correct phrasing is "This is evident when we run the test of Quadratic against Linear..."